dolphin/alert_manager/trap_receiver.py

```diff
@@ -95,6 +97,25 @@ def _extract_oid_value(var_bind):
 
         return oid, val
 
+    @staticmethod
+    def _get_alert_source_by_host(source_ip):
+        """Gets alert source for given source ip address."""
+        filters = {'host': source_ip}
+        ctxt = context.get_admin_context()
+
+        alert_source = db.alert_source_get_all(ctxt, filters=filters)
+        if len(alert_source) == 0:
+            msg = "Alert source could not be found with host %s." \
+                  % source_ip
+            raise exception.AlertSourceNotFound(message=msg)
+
+        # This is to make sure unique host is configured each alert source
+        if len(alert_source) != 1:
+            msg = "Failed to get unique alert source with host %s." % source_ip
```

Comment:
make the msg in internatinal lang. by adding _().  :)

Reply:
Done